README.md

```diff
@@ -1 +1,29 @@
-# hw01
+# Biography of Anna Kenig-Ziesler 
+
+![Image of Me](IMG_0909_smaller.jpeg)
+
+## Academic Interests
```

Comment:
Great use of headers! They seem to be formatted correctly and I like how you have two different sizes (with the # vs ##)